src/puppetlabs/puppetdb/cli/metrics_service.clj

```diff
@@ -0,0 +1,20 @@
+(ns puppetlabs.puppetdb.cli.metrics-service
```

Comment:
So the 'cli' in this namespace is meant to designate its a CLI tool, like service/import/export etc. so I'm not sure this is an appropriate place to put it. I would rather see the service in the com.puppetlabs.http.metrics namespace instead of possible, so all related items are together.